src/dlgConnectionProfiles.cpp

```diff
@@ -61,6 +61,17 @@ dlgConnectionProfiles::dlgConnectionProfiles(QWidget * parent)
     // settings suggest it:
     mudlet::self()->mShowIconsOnDialogs = !abort->icon().isNull();
 
+    auto Welcome_text_template = tr("<p><center><big><b>Welcome to Mudlet!</b><bold></center></p>"
+                                    "<p><center><b>Click on one of the games on the list to play.</b></center></p>"
+                                    "<p>To play a game not in the list, click on %1"
```

Comment:
I think @vadi2 finding the assertion being triggered on new line 104 is because the magic word (or rather `NEW_PROFILE_ICON` which gets dropped in at `%1`) does not have a space after it (so the *search and replace with an image* code cannot find it) and before the next character which is the `<` of the `<span ....>` - to fix it should be simply a matter of adding a space after the `%1` here...!

Reply:
This is HTML/Rich text - and multiple spaces get run together so the case where the word isn't used will then be:
```
                                    "<p>To play a game not in the list, click on   "
```
but those three spaces before the next characters `<span ...>` will be combined into one.